frontend/views/lobby/lobby.js

```diff
@@ -94,3 +98,26 @@ function togglePasswordInput() {
       passwordInputContainer.style.display = "none";
   }
 }
+
+const lobbyButton = document.getElementsByClassName('modalne-przycisk')[0]; 
+const lobbyName = document.getElementById('text'); 
+const isPrivate = document.querySelector('[type=checkbox]');
+const lobbyPass = document.getElementById('password-input');
+
+lobbyButton.addEventListener('click', async e => {
+  try {
+    const res = await fetch("http://localhost:3000/api/lobby", {
```

Comment:
I would move
- base address (http://localhost)
- port (3000)
- and endpoint path (/api/lobby)
to some consts/configuration file